src/arrayMethodSort.js

```diff
@@ -4,8 +4,19 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) =>
+      String(a) > String(b) ? 1 : String(a) < String(b) ? -1 : 0,
```

Comment:
Good default comparator function. It correctly handles the conversion of non-string elements to strings, as required by the Array.prototype.sort specification.